setup.py

```diff
@@ -233,7 +233,7 @@ def run(self):
     'qdarkstyle==3.0.2',
     'qstylizer>=0.1.10',
     'qtawesome>=1.0.2',
-    'qtconsole>=5.1.0',
+    'qtconsole>=5.2.0',
```

Comment:
Let's change this to `qtconsole>=5.2.0,<5.3.0` to avoid incompatibility problems in the future. You also need to do that in `requirements/conda.txt` and `binder/environment.yml`